src/Compilers/CSharp/Test/Emit/CodeGen/IndexAndRangeTests.cs

```diff
@@ -1960,5 +1549,40 @@ static string Print(Range? arg)
         }
     }
 }";
+
+        // The references we use for System.String do not have an indexer for
+        // System.Index and System.Range, so this wrapper mimics the behavior
+        private const string StringWithIndexers = @"
```

Comment:
>StringWithIndexers [](start = 29, length = 18)

Are there tests of `Index` and `Range` with `string` without this wrapper class?